.genx/prompts.js

```diff
@@ -1,24 +1,25 @@
 const apiPrompts = require('./prompts/api');
```

Comment:
In PR description, if you replace `YOUR-BRANCH-NAME` with `jamaniz/GENC-356` you'll be able to test it now.


```
rm -rf blankappseedtest && npx -y @genesislcap/genx@latest init blankappseedtest -x --ref jamaniz/GENC-356 --no-npm
```

I've published FUI fix on `@genesislcap/genx@test`, so it can be tested as follows (validation shouldn't fail):

```
rm -rf blankappseedtest && npx -y @genesislcap/genx@test init blankappseedtest -x --ref jamaniz/GENC-356 --no-npm
```